.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        ruby-version: [2.4, 2.7]
+        ruby-version: [2.7, 3.0, 3.1]
```

Comment:
> You may need to pass in MT_KWARGS_HACK=1 another way, or properly fix it. It may be that our keywords behavior is just broken.

What is this exactly and why are we using it?

Reply:
It's, as the name implies, a hack to deal with kwargs. https://docs.seattlerb.org/minitest/History_rdoc.html#label-5.16.2+-2F+2022-07-03 mentions it. I'm not too sure about the details, but as we're moving to Ruby 2.7+ we should drop it and implement proper kwargs.